containers/ddev-webserver/files/etc/passwd

```diff
@@ -1,4 +1,5 @@
 root:x:0:0:root:/home:/bin/bash
+_apt:x:100:65534::/nonexistent:/bin/false
```

Comment:
You can have two usernames with the same uid here. SO let's put _apt with uid_100. Not sure why uid 100 had group 101.